### ISSUE TITLE
`gap2` cuts wrong surface in simple "material inside bend"

### ISSUE THREAD

**Issue by mangelozzi** [shaise/FreeCAD_SheetMetal#198]
For a simple bend with `Material inside bend` with a `gap1` and `gap2` set to 10mm (all is good):
![image](https://user-images.githubusercontent.com/19764314/117356120-b0970b80-aeb3-11eb-8f65-4a26ce118d2c.png)

Now change `gap2` to 10mm: (note the strange vertical cut that appears)
![image](https://user-images.githubusercontent.com/19764314/117356224-ca385300-aeb3-11eb-849b-9ef9f4ea0a60.png)

Simple test case file here: <https://forum.freecadweb.org/download/file.php?id=152437>

Using 
```
commit e64016fb930df5d57903b1c283ce3c473d3dfb27
Author: Shai Seger <shaise@gmail.com>
Date:   Thu May 6 19:29:42 2021 +0300
    add workbench terminology
```
With PR #197 applied, and FreeCAD version:
```
OS: Windows 10 Version 2004
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 0.19.24276 (Git)
Build type: Release
Branch: releases/FreeCAD-0-19
Hash: a88db11e0a908f6e38f92bfc5187b13ebe470438
Python version: 3.8.6+
Qt version: 5.15.1
Coin version: 4.0.1
OCC version: 7.5.0
Locale: English/United Kingdom (en_GB)
```

**Comment by shaise** [comment]
should be fixed with pr #199

**Comment by mangelozzi** [comment]
> should be fixed with pr #199

Test file is working now, that was fast, thank you!!!
